app/forms/network-interface-edit.tsx

```diff
@@ -65,6 +84,69 @@ export function EditNetworkInterfaceForm({
     >
       <NameField name="name" control={form.control} />
       <DescriptionField name="description" control={form.control} />
+      <FormDivider />
+
+      <div className="flex flex-col gap-3">
+        {/* We have to blow this up instead of using TextField for better layout control of field and ClearAndAddButtons */}
+        <div>
+          <FieldLabel id="transitIps-label" htmlFor="transitIp" optional>
```

Comment:
don't think the ID is needed since `htmlFor` is there

Reply:
Agree, though `id` is currently a required prop for `FieldLabel`, and I'm not sure about side effects of making it optional. Can investigate it, though it seems minor to keep it in for now.

Reply:
Possibly related to the `aria-labelledby` magic in your comment on line 95, though not sure yet.